recipes/full_finetune_distributed.py

```diff
@@ -476,6 +476,9 @@ def train(self) -> None:
                 loss = loss / self._gradient_accumulation_steps
                 running_loss += loss
                 loss.backward()
+                grad_norm = nn.utils.clip_grad_norm_(
```

Comment:
Sorry maybe I am misunderstanding the intent of this change, but doesn't clip_grad_norm rescale the gradients? I don't think we want to do that here, right?